server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -55,6 +57,7 @@ export class ScribeLambdaFactory extends EventEmitter implements IPartitionLambd
         private readonly documentCollection: ICollection<IDocument>,
         private readonly messageCollection: ICollection<ISequencedOperationMessage>,
         private readonly producer: IProducer,
+        private readonly deltaManager: IDeltaService,
```

Comment:
Any reason why we sometimes call this `deltaManager`, and other times we call it `deltaService`?

Reply:
It seemed to me that the convention used in the repo is that we use manager whenever we are calling into the apis exposed by other microservices. Tenantmanager, GitManager. Also, there is already a deltaService that implements IDeltaService that is used in alfred to get the deltas from op storage. I thought having another deltaservice on the service side will add to the confusion

Reply:
I see. That makes sense. I got a bit confused because I had not realized/noticed that convention you mentioned, which makes sense